udata/settings.py

```diff
@@ -37,6 +37,8 @@ class Defaults(object):
     CACHE_KEY_PREFIX = 'udata-cache'
     CACHE_TYPE = 'redis'
 
+    SERVER_NAME = 'localhost:7000'
```

Comment:
I got your point.

But, if you put it in `Debug`, it is taken into account only if you run `python manage.py serve`, not if you run `inv serve`, while this is what is mentioned in the installation doc (actually, `honcho` is mentioned in the doc, but it leads the reader to user `inv serve`).

Thus, it would still not work out of the box.